src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
+    private static final Role.RoleName ROLE = Role.RoleName.USER;
+    private RoleDao roleDao;
+    private Role role;
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+        role = new Role();
+        role.setRoleName(ROLE);
+    }
+
+    @Test
+    void save_Ok() {
+        Long id = 1L;
```

Comment:
If you are comparing expected and actual values - name them:
```suggestion
        Long expectedRoleId = 1L;
```